src/Resources.resx

```diff
@@ -189,4 +189,7 @@
   <data name="The_files_to_operate_on_If_none_specified_all_files_in_workspace_will_be_operated_on" xml:space="preserve">
     <value>The files to operate on. If none specified, all files in workspace will be operated on.</value>
   </data>
+  <data name="Formatted_code_file_0" xml:space="preserve">
+    <value>Formatted '{0}'.</value>

```

Comment:
This should be changed so that the name matches the value.

Reply:
Thanks for pointing that out! I missed some text when I changed it.